### PR TITLE
adding start to budgeting

### PR DESCRIPTION
**Pull request by carriewright11**
We will definitely need help from Sandy etc. to make sure we aren't summarizing this incorrectly and to make it more understandable to people who are less familiar with grant language.

I pulled some directly from the websites and I find it a bit confusing still, but at least this gives people something to review.

### CODE REVIEW

**Comment by carriewright11** [comment]
@cansavvy  this is interesting... the url page is blank yet it says there is a broken url

**Comment by cansavvy** [comment]
> @cansavvy this is interesting... the url page is blank yet it says there is a broken url

It’s telling me ‘no broken urls’ in the comment above.

**Comment by carriewright11** [comment]
> > @cansavvy this is interesting... the url page is blank yet it says there is a broken url
> 
> It’s telling me ‘no broken urls’ in the comment above.

huh ok I guess it just didn't refresh that for me - I made an issue about this in the reports repo

**Comment by carriewright11** [comment]
I guess maybe we would just want to tell people to refresh then

File: 03-budgeting.Rmd [hunk 1]
```diff
@@ -4,14 +4,63 @@ ottrpal::set_knitr_image_path()
...
+</div>
+
+> You can refer to the [NIH Supplemental Information for Allowable Costs](https://grants.nih.gov/grants/guide/notice-files/NOT-OD-21-015.html){target="_blank"} for more information.  
+
+If you have questions, you can email: SciencePolicy@od.nih.gov
+
     
+## Unallowable costs
+
+The following costs are not allowed in the budget:
```
Comment (avahoffman) [inline 03-budgeting.Rmd:38]:
```suggestion
The following costs are **not allowed** in the budget:
```

Comment (avahoffman) [reply 03-budgeting.Rmd:38]: Some emphasis :)

File: 03-budgeting.Rmd [hunk 2]
```diff
@@ -4,14 +4,63 @@ ottrpal::set_knitr_image_path()
...
+
+  - Infrastructure costs (for example [facilities and administrative costs](https://grants.nih.gov/grants/policy/nihgps/html5/section_7/7.3_direct_costs_and_facilities_and_administrative_costs.htm), this should be indirect costs)
```
Comment (avahoffman) [inline 03-budgeting.Rmd:40]:
```suggestion
  - Infrastructure costs (for example [facilities and administrative costs](https://grants.nih.gov/grants/policy/nihgps/html5/section_7/7.3_direct_costs_and_facilities_and_administrative_costs.htm) should be indirect costs)
```

File: 03-budgeting.Rmd [hunk 3]
```diff
@@ -4,14 +4,63 @@ ottrpal::set_knitr_image_path()
...
+
+☑️  Budget justification - provide a brief summary of the DMS plan and costs- use PHS 398 Modular Budget Form: as text embedded within the Additional Narrative Justification and the R&R Budget Form: embedded within the section L. Budget Justification attachment
+
+☑️  Check that any related costs for other data policies like the [Genomic Data Sharing (GDS) Policy](https://osp.od.nih.gov/scientific-sharing/genomic-data-sharing/) are also included
+
+> See additional guidance at the [NIH Implementation Details](https://grants.nih.gov/grants/guide/notice-files/NOT-OD-22-189.html) for this policy
+
+### General tips
+
+  - Don't double up requests for direct and indirect support
```
Comment (avahoffman) [inline 03-budgeting.Rmd:58]:
```suggestion
- Don't double up requests for direct and indirect support
```

File: 03-budgeting.Rmd [hunk 4]
```diff
@@ -4,14 +4,63 @@ ottrpal::set_knitr_image_path()
...
+  - Again activities that get supported must happen during project period (for example if storing data for 10 years - must pay for those years in advance)
```
Comment (avahoffman) [inline 03-budgeting.Rmd:59]:
```suggestion
- Again activities that get supported must happen during project period (for example if storing data for 10 years - must pay for those years in advance)
```

**Review by avahoffman — APPROVED** [review]
LGTM

**Pull request merged**